### ISSUE TITLE
Settings Icons

### ISSUE THREAD

**Issue by EricBAndrews** [mlemgroup/mlem#596]
Handful of miscellaneous settings icons need updating:

- Jump button toggle should be `chevron.down.circle`
- Show website address toggle should be `link` or `link.circle`--easier to differentiate from `globe` than the current `network`
- Communities and Users options need toggles for show avatar and show banner (`person.circle` and maybe like a `person.and.background.dotted`?)

Flagging as accessibility because clear and informative icons make our *large* list of settings options easier to navigate for dyslexic users.
